packages/block-library/src/column/edit.native.js

```diff
@@ -57,6 +79,32 @@ function ColumnEdit( {
 					value={ verticalAlignment }
 				/>
 			</BlockControls>
+			<InspectorControls>
+				<PanelBody title={ __( 'Column settings' ) }>
+					<RangeControl
+						label={ __( 'Width' ) }
+						min={ 1 }
+						max={ 100 }
+						value={ columnWidths[ selectedColumnIndex ] }
+						onChange={ onWidthChange }
+						toFixed={ 1 }
+						rangePreview={
+							<ColumnsPreview
+								columnWidths={ columnWidths }
+								selectedColumnIndex={ selectedColumnIndex }
+							/>
+						}
+					/>
+				</PanelBody>
+				<PanelBody>
+					<UnsupportedFooterControl
```

Comment:
It seems exactly the same as this one - https://github.com/WordPress/gutenberg/pull/23828/files#diff-ce53a21c0c03b9b6d2e43662fa7eb866R115

Maybe it could be a separate component?